.github/workflows/docker-build.yaml

```
@@ -0,0 +1,63 @@
+      - main
+
+env:
+  PROJECT_ID: your-project-id
+  IMAGE_NAME: your-image-name
+
+jobs:
+  test-and-build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)